doc/api/buffer.markdown

```diff
@@ -112,24 +198,40 @@ console.log(buf2);
   // Prints: <Buffer 88 13 70 17>
 ```
 
-Note that when creating a Buffer using the TypeArray's `.buffer`, it is not
+Note that when creating a `Buffer` using the TypedArray's `.buffer`, it is not
 currently possible to use only a portion of the underlying `ArrayBuffer`. To
```

Comment:
@jasnell This will change if we add the `Uint8Array` signature to `Buffer.from()`.